scripts/carthage.sh

```diff
@@ -20,19 +20,20 @@ set -Eeuo pipefail
 #
 
 REPO_ROOT=$(git rev-parse --show-toplevel)
-CARTHAGE_URL="https://github.com/Carthage/Carthage/releases/download/0.39.1/Carthage.pkg"
+CARTHAGE_URL="https://github.com/Carthage/Carthage/releases/download/0.40.0/Carthage.pkg"
 CARTHAGE_DIR="$REPO_ROOT/Carthage"
-CARTHAGE="$CARTHAGE_DIR/carthage"
+CARTHAGE="$CARTHAGE_DIR/carthage-0.40.0"
```

Comment:
why this change?

Reply:
So that it actually updates carthage, otherwise the check below
```
 [[ ! -f "$CARTHAGE" ]]; then
```
would not evaluate to true even if the present version is below 0.40.0.

We can revert it if you prefer.

Reply:
no that's fine, I just wanted to understand